### ISSUE TITLE
SPO Hosted Workbench.aspx fails to load gulp-serve'd code

### ISSUE THREAD

**Issue by AJIXuMuK** [SharePoint/sp-dev-docs#4350]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

`Workbench.aspx` loads `manifests.js` using HTTPS

#### Observed Behavior

<img width="394" alt="Screen Shot 2019-07-23 at 5 27 05 PM" src="https://user-images.githubusercontent.com/17036219/61756058-25489680-ad6f-11e9-9f8b-2fc3a1c3db05.png">
<img width="974" alt="Screen Shot 2019-07-23 at 5 27 19 PM" src="https://user-images.githubusercontent.com/17036219/61756059-25e12d00-ad6f-11e9-9064-bfed9ea38023.png">

#### Steps to Reproduce

1. Create new web part (I'm using SPFx 1.9.0)
2. `gulp serve --nobrowser`
3. open `layouts/15/workbench.aspx`

**Comment by AJIXuMuK** [comment]
More details:
It looks like it tries to load the file twice.
First time using https and it is actually loaded.
But there there is some error and second request with http.

<img width="1669" alt="Screen Shot 2019-07-23 at 5 34 50 PM" src="https://user-images.githubusercontent.com/17036219/61756322-4a89d480-ad70-11e9-86f4-eaf62f67ce00.png">

<img width="962" alt="Screen Shot 2019-07-23 at 5 36 14 PM" src="https://user-images.githubusercontent.com/17036219/61756348-64c3b280-ad70-11e9-923b-0196893c53f9.png">

**Comment by AJIXuMuK** [comment]
Setting `"https": false` in `serve.json` doesn't help.

**Comment by martinlingstuyl** [comment]
Same here... I'm back to re-installing 1.8.2 on my project

**Comment by mmohamedbasith** [comment]
i am facing the same issue .  @VesaJuvonen

**Comment by FAT77** [comment]
Same problem. Since upgrade to 1.9.0 hosted workbench stopped working.

**Comment by jcosta33** [comment]
Same problem with extensions...

**Comment by patmill** [comment]
Looking...

**Comment by patmill** [comment]
Also - what does the command prompt look like when gulp is running?  What files are being requested, etc.

**Comment by mmohamedbasith** [comment]
@patmill i can confirm that  I tried with react hello world project (without adding any addition code). The issue occurred.  i dont have screenshot now any as i have reverted my SPFX back to 1.8.2 again after a long days struggling with 1.9.0 version

**Comment by patmill** [comment]
OK, I have found a tenant where this repros, and the same project, same code, succeeds in another tenant.  They are both running the same production code as well.  So on the plus side, at first glance this looks like a problem on our servers, not with the npm bits.  Stay tuned.

**Comment by AmonReviled** [comment]
@patmill Yes, the issue occurs with a plain, no-framework project.
![image](https://user-images.githubusercontent.com/4827728/61815365-b53e1d00-ae0f-11e9-9683-1e348382a0d2.png)
![image](https://user-images.githubusercontent.com/4827728/61815519-05b57a80-ae10-11e9-9c1b-2f0c39142108.png)
![image](https://user-images.githubusercontent.com/4827728/61815391-c424cf80-ae0f-11e9-820d-759c8f9412f0.png)
![image](https://user-images.githubusercontent.com/4827728/61815431-d9016300-ae0f-11e9-9185-5b67d43b1993.png)

**Comment by patmill** [comment]
Yeah, so the errors in manifests.js are a red herring.  The problem is that ultimately the parsing/js loading of manifests.js doesn't happen in the error case, because the script element doesn't wind up in the DOM.  Digging in further.

**Comment by patmill** [comment]
OK - so we have found the problem, and have a solution.  We need to figure out the best way to publish it.  There was a change in how the local manifests.js file is built, and it caused an issue on the hosted workbench.

Unfortunately, our release process winds up using a codepath that bypassed this error.  We've updated the steps to ensure this doesn't happen again.  Will post here when we have next steps.
